concourse/pipelines/guest-package-build.yaml

```diff
@@ -149,40 +149,60 @@ jobs:
           source-image: projects/debian-cloud/global/images/family/debian-9
           dest-image: debian-9-((.:build-id))
           gcs-package-path: gs://gcp-guest-package-uploads/guest-agent/google-guest-agent_((.:package-version))-g1_amd64.deb
+          machine-type: e2-medium
+          worker-image: projects/compute-image-tools/global/images/family/debian-10-worker
       - task: build-package-image-debian-10
         file: guest-test-infra/concourse/tasks/daisy-build-package-image.yaml
         vars:
           source-image: projects/debian-cloud/global/images/family/debian-10
           dest-image: debian-10-((.:build-id))
           gcs-package-path: gs://gcp-guest-package-uploads/guest-agent/google-guest-agent_((.:package-version))-g1_amd64.deb
+          machine-type: e2-medium
```

Comment:
just a note, having to specify the machine type and not being able to set a default is a limitation of separated task files, and goes away when we move this pipeline to a jsonnet template.